source/documentation/managing_apps/autoscaling.erb

```diff
@@ -0,0 +1,278 @@
+## Autoscaling
+
+Apps can be configured to dynamically scale in and out based on autoscaling
+policies.
+
+Scaling your app to fewer instances can reduce the cost during periods of low
+usage,
+eg at night.
+
+Scaling your app based on throughput or average latency can improve the user
+experience of your app during periods of high traffic,
+eg before a submission deadline.
+
+### Scaling methods
+
+There are two ways of scaling:
+
+1. `schedules` - scale apps based on time of day and day of week or month
+1. `scaling_rules` - scale apps based on metrics, eg average latency
+
+Schedules and scaling rules can be used at the same time, or on their own.
+
+### Limits
+
+There are two limits which you can apply to an autoscaling policy:
+
+* `instance_max_count` - maximum number of instances to run
+* `instance_min_count` - minimum number of instances to run
+
+### Metrics
+
+You can scale your app based on some provided metrics:
+
+* `memoryused` - how many megabytes of memory an app instance is using
+* `memoryutil` - percentage of memory utilization where `50` is 50%
+* `cpu` - percentage of CPU utilization where `100` is 100% of a single CPU
+* `responsetime` - average time in milliseconds an app instance takes to respond to external HTTP requests
+* `throughput` - number of external HTTP requests processed per second
+
+You can also scale your app based on custom metrics, please refer to the
+[app-autoscaler user guide](https://github.com/cloudfoundry/app-autoscaler/blob/develop/docs/Readme.md#auto-scale-your-application-with-custom-metrics)
+on GitHub.
+
+### Example
+
+1. Deploy your app, where `APPNAME` is the name of your application:
+
+    ```
+    cf push APPNAME
+    ```
+
+2. Create an autoscaler service to scale your app:
+
+    ```
+    cf create-service autoscaler autoscaler-free-plan scale-APPNAME
+    ```
+
+3. Bind the autoscaler service to your app:
+
+    ```
+    cf bind-service APPNAME scale-APPNAME
+    ```
+
+4. <p>Create an autoscaling policy, the following policy will scale your app:<ul>
+   <li>adding one instance if CPU usage is greater than 50% for 120 seconds</li>
+   <li>removing one instance if CPU usage is less than 50% for 120 seconds</li>
+   </ul></p><p>It will ensure:<ul>
+   <li>there are never more than 5 instances of your app</li>
+   <li>there is never fewer than 1 instance of your app</li>
```

Comment:
Should this be 2? Otherwise, I can see people taking it as a recommendation and then getting upset when cells are rolled. Also to apply to below code block.

Reply:
This is an example detailing how policies work, so this should be one, because the policy has 1 (unless we want to change this example)

In the `Tips for autoscaling policies` section there is:
> If your app should be highly available, ensure `instance_min_count` is at least 3

Reply:
Is it worth duplicating that at the top to make it really obvious?

Reply:
Done in 50109a0

Reply:
:+1: